### ISSUE TITLE
Plugin seems to have stopped loading loras

### ISSUE THREAD

**Issue by JasonS09** [Acly/krita-ai-diffusion#375]
Hello. Been using the plugin for a while. I use a Google Colab instance to run ComfyUI. Today, I have been attempting to use one of my saved styles, but judging by the result it has been stopped loading loras.

This is my current config, using one of my Rocket Raccoon trained loras:

![Screenshot 2024-02-01 140257](https://github.com/Acly/krita-ai-diffusion/assets/47617351/45bba23d-3d19-4b1c-aa79-d51b250644a9)
![Screenshot 2024-02-01 135415](https://github.com/Acly/krita-ai-diffusion/assets/47617351/361d8347-ff88-427f-8b4e-21e06de65dff)

I get the following result:

![RocketTest](https://github.com/Acly/krita-ai-diffusion/assets/47617351/9bc81de2-7f8b-4935-ac40-4919a6437e90)

This what I get by running the following workflow in ComfyUI:

![Screenshot 2024-02-01 135617](https://github.com/Acly/krita-ai-diffusion/assets/47617351/8f62fb63-611b-4c12-be14-c7e35af09976)
![ComfyUI_00074_](https://github.com/Acly/krita-ai-diffusion/assets/47617351/62216638-9f48-4f9f-af1d-2eff964d1635)

Which is the kind of result I would expect from this workflow.
Using Comfy, this is what I get if I remove all the loras.
![ComfyUI_00075_](https://github.com/Acly/krita-ai-diffusion/assets/47617351/68329f5b-88e1-4195-b6f5-8785e233916a)

Which looks more similar to what I get using Krita.

**Comment by JasonS09** [comment]
> Hm, Lora still work for me in general. Since last release there is a "Dump Workflow" option in Interface settings. This will write the Comfy prompt to the logs folder, you can load it in ComfyUI and check if the Lora nodes are there.

I just checked. Indeed they aren't present in the workflow for some reason.

[workflow.json](https://github.com/Acly/krita-ai-diffusion/files/14141774/workflow.json)

**Comment by Acly** [comment]
Ah probably you run Krita on Windows, but Comfy runs on Linux (via Colab). I notice the Lora are in subfolders, so I think it's #307
There should be a warning in client.log that the Lora isn't found (maybe this should be an error...) because it uses the wrong path separator.

**Comment by JasonS09** [comment]
Yeah, that's more likely the problem. The errors are also there, I just checked.
